desktop/core/src/desktop/conf.py

```diff
@@ -1818,6 +1818,16 @@ def get_instrumentation_default():
   default=False
 )
 
+def get_chunked_fileuploader():
```

Comment:
Directly use the config wherever required?

Reply:
+1 on using the config directly. But, even otherwise the function name doesn't sound right. You are not returning `chunked_fileuploader` object. If this function returns a boolean for enabled/disabled that it should be named accordingly.

Reply:
I tried using the 'ENABLE_NEW_FILE_UPLOADER.get()' config directly but seems like its not initialized when the hue server is starting. 

I get the following error:
`/settings.py", line 657, in <module>
    if ENABLE_NEW_FILE_UPLOADER.get():
AttributeError: 'Config' object has no attribute 'get'`

So I'm using something similar to [desktop.conf.is_custom_jwt_auth_enabled()](https://github.com/cloudera/hue/blob/9c61e7a16c6cab7ac1e929d7a8b3c242cdd23172/desktop/core/src/desktop/settings.py#L334)

@amitsrivastava changed the function's name to 'is_chunked_fileuploader_enabled()'